Sources/Assets.xcassets/TopPrgressviewbg.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "TopPrgressviewbg.svg",
```

Comment:
name it appropriately, topBackgroundSlider, bottomBackgroundSlider, something like that?